api/src/migrations/add-branding-doc.js

```diff
@@ -26,7 +26,7 @@ const attachDocument = (src, resource) => {
       }
       resolve({
         content_type: resource.type,
-        data: Buffer.from(data, 'base64')
+        data: Buffer.from(data).toString('base64')
```

Comment:
I think you've changed the behaviour here - `Buffer.from` returns a `Buffer`, while `Buffer.toString` returns a `String`. You've also changed the encoding used to create the buffer from "base64" to "utf" (default) which I think will cause it not to work.

I think the previous implementation should pass the eslint rules so I don't think you need to modify this file at all, or am I missing something?

Reply:
That's right, I have keenly gone through all instances where `Buffer` is being used and set the right behaviour

Reply:
I don't follow. I'm saying your code is wrong, and the original code was right. Please put it back the way it was.